charts/posthog/values.yaml

```diff
@@ -95,6 +95,8 @@ web:
     # requests:
     #   cpu: 300m
     #   memory: 300Mi
+  # -- Location from which application static assets (js, css, etc.) should be served
+  staticUrl: "/static/"
```

Comment:
staticAssetsUrl?

Reply:
Yeah I was mirroring the django env var, but staticAssetsUrl is a bit clearer out of django context.